modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2712,6 +2713,9 @@ category-display-page-help=The display page is used to display the category.
 category-display-pages=Category Display Pages
 category-facet-display-context=Category Facet Display Context
 category-facet-display-context-help=Access the data contained in the AssetCategoriesSearchFacetDisplayContext object (for example, isNothingSelected returns true if terms have not been selected).
+category-facet-field=Category Facet Field
+category-facet-field-configuration-name=Category Facet Field
+category-facet-field-help=Select the field to use in the Category Facet Widget.
```

Comment:
@BryanEngler This needs more explanation. /cc @rbohl

Reply:
I think I need a bit of clarification on this one. What does it do? I wasn't aware you could select a filed in the category facet and I can't think of why you'd need to for the new behavior. @lipusz

Reply:
@rbohl This is for an Instance/System Settings entry with an option that allows the admin to configure which index field `assetCategoryIds` (current) or `assetVocabularyCategoryIds` (new) to be used for aggregation. 

The "and/or" operator and the "Vocabulary Scope" new features (correct me if I'm wrong @BryanEngler) will work with the new field only.

Reply:
More context: https://liferay.slack.com/archives/C03LFD93BPF/p1662733042084449

Reply:
Is this too long or unclear?
> Choose whether the Category Facet aggregates documents by the assetVocabularyCategoryIds field or the assetCategoryIds field (default). To view categories hierarchically by vocabulary, the Category Facet must use the assetVocabularyCategoryIds field.

Reply:
im not sure about the parenthetical text... depending on the upgrade path, assetVocabularyCategoryIds could be the default... maybe "legacy" is a good term to use here after assetCategoryIds

 > To view categories hierarchically by vocabulary

this is technically incorrect, if we said 

> To filter categories by vocabulary

it would probably be more accurate

but in my opinion the "real" help text should be in the widget config and it would point to this config, so to complete that loop maybe we should mention something here that this config makes additional options in the category widget available and kind of just points back to there...

Reply:
That sounds fine to me. Then we could use

> Selecting assetVocabularyCategoryIds enables additional configuration options in the Category Facet's widget configuration. Learn More.

I'm guessing you'll have an identical field selector in the widget configuration. If so the widget configuration help text could be

> Choose whether the Category Facet aggregates documents by the assetVocabularyCategoryIds field or the assetCategoryIds field. To enable filtering of categories by vocabulary use the assetVocabularyCategoryIds field.

Am I still off?

Reply:
there wont be a selector in the widget config, and we decided that the help text in the widget will be 
>To enable this configuration follow the steps in the documentation. Learn More.

so i think for this one, 
>Selecting assetVocabularyCategoryIds enables additional configuration options in the Category Facet's widget configuration. Learn More.

is good, and both "Learn More" links will point to the same article